Adyen/UI/Form/Items/Text/FormTextItemView.swift

```diff
@@ -299,12 +302,21 @@ open class FormTextItemView<ItemType: FormTextItem>: FormValueItemView<String, F
             hideAlertLabel(false)
             highlightSeparatorView(color: item.style.errorColor)
             titleLabel.textColor = item.style.errorColor
+        } else {
+            if case .customView = accessory {
```

Comment:
This is to ignore "required" empty fields while editing, right?

Reply:
this is because we validate fields also on creation now. but if the field has an accessory view (scheme logo) there is custom logic involved. so we need to check whether it has a `.customView` as an accessory

Reply:
I mean this whole `else { ... }` part 😅

Reply:
this is to properly handle validation of empty values and be able to revert to the initial state.